eng/scripts/Fetch-PackageList-Javadoc.ps1

```diff
@@ -0,0 +1,51 @@
+<#
+  .SYNOPSIS
+  Fetch the package list from javadoc jar and checkin to docs/metadata
+
+  .DESCRIPTION
+  The Verify-Links.ps1 script will check whether the files contain any broken links.
+
+  .PARAMETER JavaDocJarLocation
+  Specifies the file that contains a set of links to ignore when verifying.
+
+  .PARAMETER DocRepoLocation
+  Specifies the file that contains a set of links to ignore when verifying.
+
+  .PARAMETER ArtifactName
+  Specifies the file that contains a set of links to ignore when verifying.
+#>
+[CmdletBinding()]
+param (
+  [string] $JavaDocJarLocation = "",
+  [string] $DocRepoLocation = "",
+  [string] $ArtifactName = ""
+)
+. (Join-Path $PSScriptRoot ../common/scripts/common.ps1)
+
+function Fetch-Namespaces-From-Javadoc ($jarFilePath, $tempLocation, $destination) {
```

Comment:
The `$destination` parameter is not used and it looks like you're referring to `$packageNameLocation` which is declared in an outside scope. 

I think `$packageNameLocation` should probably be the `$destination` passed to this function.